src/calculateRentalCost.js

```diff
@@ -3,8 +3,20 @@
  *
  * @return {number}
  */
-function calculateRentalCost(days) {
-  // write code here
+function CalculateRentalCost(days) {
+  const dailyCost = 40;
+
+  let totalCost = days * dailyCost;
```

Comment:
Consider using `const` instead of `let` for `totalCost` since it is not reassigned after its initial value is set.